src/audio/multiband_drc/multiband_drc.h

```diff
@@ -21,6 +21,12 @@
 /**
  * Stores the state of the sub-components in Multiband DRC
  */
+struct multiband_drc_coefficients {
+	struct sof_eq_iir_biquad crossover[SOF_MULTIBAND_DRC_MAX_BANDS * PLATFORM_MAX_CHANNELS];
+	struct sof_eq_iir_biquad emphasis[SOF_MULTIBAND_DRC_MAX_BANDS * PLATFORM_MAX_CHANNELS];
+	struct sof_eq_iir_biquad deemphasis[SOF_MULTIBAND_DRC_MAX_BANDS * PLATFORM_MAX_CHANNELS];
+} __packed;
```

Comment:
Won't packing cause a lot of slow reads/writes because things might not be aligned?

Reply:
While check-in I was getting the error if I do  `__attribute__((packed));`  instead of "`} __packed;` so I drop the attribute.

"
```
No codespell typos will be found - file '/usr/share/codespell/dictionary.txt': No such file or directory
WARNING: Prefer __packed over __attribute__((packed))
#10: FILE: src/audio/multiband_drc/multiband_drc.h:28:
+} __attribute__((packed));

total: 0 errors, 1 warnings, 8 lines checked

```


Understood. We're using `__packed` to save memory, but could you confirm if the potential misalignment outweighs the memory gain in our context? And whether our target platform can handle unaligned accesses efficiently.

Reply:
@ShriramShastry we mainly using packing for any data shared with host, but in this case isn't this data provate to FW only ? If so, we can drop the packed and allow compiler to best organise the data.

Reply:
agree, `packed` isn't needed. It should only be used where data format is important and must be preserved, e.g. when sending data over networks, or between the host and the DSP

Reply:
Done, removed `packed `